.ci/openvino-onnx/Jenkinsfile

```
@@ -1,7 +1,9 @@
-
 // Copyright (C) 2018-2020 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 
+DOCKER_CONTAINER_NAME="jenkins_ngraph-onnx_ci_${BUILD_NUMBER}"
```

You can unify the names...
```suggestion
DOCKER_CONTAINER_NAME="openvino-onnx-ci_${BUILD_NUMBER}"
```